examples/main-auth.c

```diff
@@ -145,7 +145,7 @@ int main(int argc, char *argv[])
 	fprintf(stderr, "jwt decoded successfully!\n");
 
 	/* Validate jwt */
-	if (jwt_validate(jwt, jwt_valid) <= 0) {
+	if (jwt_validate(jwt, jwt_valid) > 0) {
```

Comment:
This should actually be `!= 0` The documentation in `jwt.h` says `0 on success`

Reply:
:+1: Resolved.